### ISSUE TITLE
Need some guidelines 

### ISSUE THREAD

**Issue by MEhtisham8217** [keras-team/keras#14289]
Use the same dataset (LFW )
Apply a pretrained convolutional neural network, (using VGG16), and replace the fully-connected layers with your own. Freeze the weights of the convolutional layers and only train the new FC layer.
Sample code for using pre-trained VGG16 for another classification task is available from:
https://github.com/keras-team/keras/issues/4465

Which Code should i use?

**Comment by yash-clear** [comment]
`from keras.applications.vgg16 import VGG16`
`from keras.models import Model`
`from keras.layers import Dense,Flatten`
`# load model without classifier layers`
`model = VGG16(include_top=False, input_shape=(500, 500, 3))`
`#adding new layers`
`flattened = Flatten()(model.layers[-1].output)`
`layer1 = Dense(1024, activation='relu')(flattened)`
`output = Dense(10, activation='softmax')(layer1)`
`#new model`
`model = Model(inputs=model.inputs, outputs=output)` 
`model.summary()`

**Comment by sushreebarsa** [comment]
@MEhtisham8217 Could you please refer to this [link](https://keras.io/api/applications/vgg/) and let us know if it helps ?Thanks!

**Comment by MEhtisham8217** [comment]
> @MEhtisham8217 Could you please refer to this [link](https://keras.io/api/applications/vgg/) and let us know if it helps ?Thanks!

Yeah i did that months ago thanks anyway

**Comment by sushreebarsa** [comment]
@MEhtisham8217 Thanks for the update,could you please let us know if it is good to close this issue? Thanks!
